app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1142,6 +1142,49 @@ class PatientRepositoryAndroidTest {
     }
   }
 
+  @Test
+  fun verify_deleted_Appointments_are_not_included_when_fetching_recent_patients() {
+    val facilityUuid = UUID.randomUUID()
+    val recentPatient1 = savePatientWithAppointment(facilityUuid = facilityUuid)
+    savePatientWithAppointment(facilityUuid = facilityUuid, deletedAt = Instant.now())
```

Comment:
Can you assign variable to the `savePatientWithAppointment` method call?

Reply:
done